meteor-app/imports/ui/service/clientOP2.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-lone-blocks */
+import React from 'react';
+import { useForm } from 'react-hook-form';
+// import * as yup from "yup";
+
+
+
+
+{/*
+
+considering running the below on an array of data to make the component/form extensible
+
+function inputElement(type, name, placeholder){
+  return(
+  <label className="service-form-label">
+  <input type={type} name={name} ref={register({required: true, message: `Please confirm the ${placeholder} `})}  className="service-form-input"/>
+        <span className="placeholder"> {placeholder}</span>
+  </label>
+  )
+  {errors.{name} && errors.{name} === "required" (<p>{errors.{name}.message}</p>)}
+}
+
+
+  const customerQuestions = SOMETHING.customer.map(inputElement(type, name, placeholder){
+    return <inputEle type={type} name={name} placeholder{placeholder} key={name}/>
+  })
+
+  const bikeQuestions = SOMETHING.bike.map(inputElement(type, name, placeholder){
+    return <inputEle type={type} name={name} placeholder{placeholder} key={name}/>
+  })
+
+
+*/}
+
+
+function FormTrial() {
+  const { register, handleSubmit, errors} = useForm({
```

Comment:
I would prefer to use proper names for components, ie provide some kind of information as to what the purpose of this form is. Remember that the component importing this file can choose a different name for it

Reply:
This is solved by deleting the file I think.